unit_tests/src/test_rcMisc_functions.cpp

```diff
@@ -0,0 +1,304 @@
+#include "catch.hpp"
```

Comment:
Maybe it's a little overboard, but I would think that this test file is specific to delay directive options. I mainly want to avoid referring to rcMisc in any way :) Thoughts?

Reply:
Sure. We can change the name. I couldn't decide on one at the time. We'll need to change it in the main/4.3 branch if we change it here.

Reply:
changed it.